tests/manage/mcg/helpers.py

```diff
@@ -152,3 +161,136 @@ def upload_parts(mcg_obj, awscli_pod, bucketname, object_key, body_path, upload_
         ).split("\"")[-3].split("\\")[0]
         parts.append({"PartNumber": count, "ETag": f'"{part}"'})
     return parts
+
+
+def oc_create_aws_backingstore(cld_mgr, backingstore_name, uls_name, region):
```

Comment:
I wouldn't add creation of backingstores into helpers. I would add it as a new resource class into `ocs-ci/ocs_ci/ocs/resources/backingstore.py`

Reply:
This suggestion is a nice idea but since this pr needs to be pushed to allow further work in other areas I've opened an issue with a link to this pr so we can implement it in a different time:
https://github.com/red-hat-storage/ocs-ci/issues/1854